src/app/+my-dspace-page/my-dspace-page.component.ts

```diff
@@ -126,7 +128,10 @@ export class MyDSpacePageComponent implements OnInit {
     this.searchOptions$ = this.searchConfigService.paginatedSearchOptions;
     this.sub = this.searchOptions$.pipe(
       tap(() => this.resultsRD$.next(null)),
-      switchMap((options: PaginatedSearchOptions) => this.service.search(options).pipe(getSucceededRemoteData())))
+      switchMap((options: PaginatedSearchOptions) =>
```

Comment:
I think that this is not the right place where to follow links for claimed task, because this search is used for both workspaceitem and workflowitem and the first ones don't have a link to `workflowitem`. I think the right places are the specific search result classes for pooltask and claimedtask (`ClaimedSearchResultListElementComponent`, `PoolSearchResultListElementComponent` etc)